### ISSUE TITLE
Add support to retrieve measurements

### ISSUE THREAD

**Issue by bradgarropy** [coddingtonbear/python-myfitnesspal#14]
# Why
Some **MyFitnessPal** users like myself are interested in calculating how their body metrics are changing on a daily, weekly, or monthly basis. Measurements like weight and body fat are particularly important to track while bulking or cutting in order to gauge weight changes week over week.

# What
I would like to add support to retrieve measurements from **MyFitnessPal**. Measurement names are defined by the user and their values are stored accordingly based on the date. You can view measurements on the page pictured below.

![measurements](https://cloud.githubusercontent.com/assets/11336745/7290728/d8c220e4-e949-11e4-8f9a-3eac0e38ac51.PNG)

# How
Measurements can be scraped from the **MyFitnessPal** URLs in this format:
http://www.myfitnesspal.com/measurements/username/edit?page=1&type=1

The *page* value can be altered to scroll through dates and the *type* value can be altered based on which measurement you want to view. User defined measurements, like *body fat percentage* for example, come with a type that looks to be a hashed value of the name. These are listed in the HTML like so:

```html
<select class="select" id="type" name="type">
<option value="1" selected="selected">Weight</option>
<option value="90533147">Body Fat</option>
<option value="92749113">Bicep</option>
<option value="92749117">Chest</option>
<option value="92749121">Waist</option>
<option value="92749122">Shoulders</option></select>
```

Based on this information we can correlate the measurement name with the *type* value. The measurements and values are in the HTML in a very straightforward manner.

```html
<tr>
    <td class="first">
        Weight
    </td>
    <td class="col-num">4/22/2015</td>
    <td class="col-num">173.2 lbs</td>
    </td>
    <td>
        <a href="/measurements/delete/169726739" class="remove" data-confirm="Are you sure you want to delete this entry?">Delete?</a>
    </td>
</tr>
```

# Notes
If you are in agreement about adding this feature, I will fork this repository with a branch name of **add-measurement-support**, and start developing!

I would like to hear your feedback about everything, and especially these points.
* Extend the `client.get_date` method or create a new `client.get_measurement` method.
* The data structure that will contain measurement information.

**Comment by coddingtonbear** [comment]
+1 sounds good to me so far!  Maybe you'd like to hop onto freenode (I'm `coddingtonbear` there, too) and send me a message; we might want to talk a little bit about how to architect it.

**Comment by bradgarropy** [comment]
Closed by #15.
